web/main/templates/export/as_printable_html/casebook.html

```diff
@@ -58,9 +58,11 @@ <h2 class="casebook subtitle">{{ casebook.subtitle|default_if_none:""}}</h2>
         </ul>
       </div>
       <section class="casebook headnote">{{ casebook.headnote|safe}}</section>
+      {% if page.number == 1%}
```

Comment:
Only displays the TOC markup on the first section; previously this was just hidden by CSS but I wanted to add a page break before it, and I couldn't get PagedJS to ignore it even if it was hidden.